cli/server_run_command.go

```diff
@@ -160,21 +159,7 @@ func (c *SrvRunCmd) getRandomPort() (string, error) {
 }
 
 func (c *SrvRunCmd) dataParentDir() (string, error) {
```

Comment:
we can probably just remove this function entirely and call the real one where needed?

Reply:
Certainly possible. I had just wanted to preserve the semantic meaning of having a "data parent dir" while delegating the implementation details to the precise meaning of `xdgShareHome()`, but it's likely never going to change since it's already a solid convention.